website/content/en/highlights/2023-02-28-0-28-0-upgrade-guide.md

```diff
@@ -23,3 +27,18 @@ We cover them below to help you upgrade quickly:
 
 The `units` option had been replaced by `include_units` in the `0.10` release. This release marks the final removal
 of this option, please update your configurations to use the `include_units` option instead.
+
+### Potentially impactful changes
+
+#### AWS components now use OpenSSL as the TLS implementation {#aws-openssl}
+
+In this release, Vector's AWS integrations had their TLS implementation swapped from
+[`rustls`][rusttls] to OpenSSL. We don't expect any user visible impacte,
```

Comment:
```suggestion
[`rustls`][rusttls] to OpenSSL. We don't expect any user visible impact,
```

Reply:
Unless this is fancy spelling I'm not aware of 😉

Reply:
Need `check-spelle` 😂